### PR TITLE
refactor(schematics): remove `CommonModule`

### PR DESCRIPTION
**Pull request by JeanMeche**
The template now uses the new control flow syntax

### CODE REVIEW

File: packages/schematics/angular/application/files/standalone-files/src/app/app.component.ts.template
Comment (eneajaho) [inline packages/schematics/angular/application/files/standalone-files/src/app/app.component.ts.template:1]: Can we also do 

```html
<router-outlet />
```

**Review by JoostK — COMMENTED** [review]
I'll defer as to whether this change is desirable, as it does also mean that common pipes are no longer available by default.

File: packages/schematics/angular/application/files/standalone-files/src/app/app.component.ts.template [hunk 1]
```diff
@@ -5,12 +5,12 @@ import { RouterOutlet } from '@angular/router';<% } %>
 @Component({
   selector: '<%= selector %>',
   standalone: true,
-  imports: [CommonModule<% if(routing) { %>, RouterOutlet<% } %>],<% if(inlineTemplate) { %>
+  imports: [<% if(routing) { %>RouterOutlet<% } %>],<% if(inlineTemplate) { %>
```
[the remark this answers is not in the document]
Comment (alan-agius4) [reply packages/schematics/angular/application/files/standalone-files/src/app/app.component.ts.template:8]: I am a bit split here too, but AFAIK importing `CommonModule` is not also the recommended/best approach. As it's preferred to import the individual APIs.

**Comment by clydin** [comment]
Looks like there are some test failures here.
One of them appears related to the router outlet change.

**Review by alan-agius4 — APPROVED** [review]
LGTM

**Pull request merged**